storage/innobase/log/log0recv.cc

```diff
@@ -3857,7 +3857,7 @@ bool meb_scan_log_recs(
      * check redo_log.cc read_log_seg_pre8030 & scan_log_recs_pre8030.
      */
     if (scanned_lsn + data_len > recv_sys->scanned_lsn) {
-      if (xtrabackup_original_log_format >= Log_format::VERSION_8_0_30 &&
+      if (log_detected_format >= Log_format::VERSION_8_0_30 &&
```

Comment:
just minor suggestion. To maintain clarity that variable is introduced by pxb, we should use xb_ prefix.
xb_log_format or xb_log_detected_format?

Then seeing it any place we know that it our variable. Even if it is in server code.

We can enforce this rule for new vars that go around and sit in innodb/server code.

As of now, this is optional.

Reply:
btw, I was the one who introduced this var. Didn't give much thought back then

Reply:
Thanks